### DISCUSSION TITLE
Having a helluva time program the Anytone D878UVIII Plus

### DISCUSSION THREAD

**Discussion by kaunix** [hmatuschek/qdmr#230] [Q&A]
I've tried this under linux and Mac OS. my radio came pre-programmed with a code plug. I can download it just fine. but when I tried to modify the codeplug and push it back up to the radio qdmr throws a bunch of errors about doesn't like MAX setting in channel list, it must have at least 1 contact (even though there are several in the plug. etc...

did I not see something in the discussions that resolve this? I'm DMR stupid but linux and MacOS savvy and I've programmed other non-DMR radios on these platforms before.

help please?

**Comment by kaunix** [comment]
I've also booted the radio back to factory defaults and gotten the same errors. 

![Verify Codeplug 2022-05-25 12-16-42](https://user-images.githubusercontent.com/30199459/170359733-57c5fde9-4926-430b-b252-bb280c2dc894.png)

**Comment by hmatuschek** [comment]
Can you send me the firmware version you are using? I have to check if the codeplug has changed since version 0.8.0. This is actually quiet likely, as AnyTone is releasing new radios and firmware versions faster than I can update qdmr. If I remember it correctly, it should be version 2.01b.

**Reply by kaunix** [reply]
Model Name: D878UVII
Firmware Ver: 2.05
Hardware Ver: 1.10(GD)

**Reply by hmatuschek** [reply]
Ok, thanks. That is considerably newer than the 2.01 I used to reverse engineer the codeplug from. I'll update the codeplugs for the anytone devices soon anyway, and I will update that one too. It is somewhat hard to keep up with the anytone devices and firmware updates.

**Comment by JebBaker** [comment]
This is a little old...but...

I just purchased an 878 and wanted to see what the progress is
Thank you

**Reply by peterloron** [reply]
Also following! The current firmware rev is 3.0.1.

**Reply by allesand** [reply]
878 UVII Plus here with firmware 3.01 - qdmr is reading and writing the configuration just fine, at least those parameters it supports. The messages in the above screenshot are mainly: power level in channel configuration (that should be fixed in recent qdmr versions) and frequency out of range - there is a option regarding that in the qdmr settings. In case you are using Linux: depending on your distribution, the qdmr version might be quite old. https://dm3mat.darc.de/qdmr/install.html has instructions on getting a more recent version. You might want to give it a try with a recent version of qdmr?
